fdbclient/ClientWorkerInterface.h

```diff
@@ -88,4 +90,41 @@ struct ProfilerRequest {
 	}
 };
 
+struct SetFailureInjection {
+	constexpr static FileIdentifier file_identifier = 15439864;
+	ReplyPromise<Void> reply;
+	struct ThrottleDiskCommand {
+		// how often should the delay be inserted (0 meaning once, 10 meaning every 10 secs)
```

Comment:
Why does 0 mean 1?  Injecting a single delay does not seem like a useful action.  I think it would make more sense for 0 to mean off / stop injecting delay.

Reply:
Updated based on our discussion